test/chrome/script.py

```diff
@@ -0,0 +1,58 @@
+# Run Chromium tests for HTTPS Everywhere
+#
+# This script may be executed as `python script.py [directory of CRX]`
+#
+# The script is compatible with Python 2. Python 3 is not tested.
+# Selenium, WebDriver and Google Chrome (or Chromium) must be installed
```

Comment:
Can you include a requirements.txt that install Selenium? Is WebDriver a Python package or ... ? If the former, include it in the requirements.txt.